tests/integration/security/pass_through_filter_chain_test.go

```diff
@@ -37,10 +37,12 @@ func TestPassThroughFilterChain(t *testing.T) {
 		Features("security.filterchain").
 		Run(func(ctx framework.TestContext) {
 			ns := apps.Namespace1
+
 			type expect struct {
-				port   int
-				schema protocol.Instance
-				want   bool
+				port      int
+				schema    protocol.Instance
+				plaintext bool
```

Comment:
it feels a bit confusing there are both `plaintext` and `mtls` for the expected outcome, can we just have one?

Reply:
Its meant to be `expectedOutcomeFromPlaintextRequest` and `expectedOutcomeFromMTLSRequest`. I can rename it to something between that and the current code?